src/PlayableCharacter.java

```diff
@@ -0,0 +1,123 @@
+public class PlayableCharacter {
+    private String name;
+    private int level;
+    private int experience;
+    private int experienceToLevelUp;
+    private String item;
+    private int maxHP;
+    private int HP;
```

Comment:
Keep in mind that we generally name variables using lower camelcase. UPPERCASE is reserved for constants. 
```suggestion
    private int hp;
```